syncto/__init__.py

```diff
@@ -32,6 +35,9 @@ def main(global_config, **settings):
                        default_settings=DEFAULT_SETTINGS)
 
     settings = config.get_settings()
+    config.registry.heartbeats['sync'] = get_token_server_ping(
+        settings['token_server_url'],
+        settings['token_server_heartbeat_timeout_seconds'])
```

Comment:
the functions expected in heartbeats are simple functions with a request argument. Here you made a sort of closure on settings, and I don't see any obvious reason to do so ?